native/machine-coding/Quiz-app-with-timer/index.css

```diff
@@ -0,0 +1,103 @@
+@import url(../../helpers/core.css);
```

Comment:
please remove this as per the new standards introduced

Reply:
do i need to readd * rule that was previously used?

Reply:
nevermind i understood.